README.md

```diff
@@ -1 +1,229 @@
+
+### Key Features <a name="key-features"></a>
+
+
+
+- **[Linters Setup]**
+- **[Follow HTML and CSS best practice]**
+- **[Display text]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+- [Live Demo Link](https://vikas924.github.io/Hello-Microverse/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+- Visual Studio Code.
+- Node JS.
+- Git bash.
+- GitHub Account.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Use git clone command or download ZIP folder
+
+```sh
+  cd my-folder
+  git@github.com:vikas924/Hello-Microverse.git
+```
+
+
+### Install
+
+Install this project with:
+
+npm
+
+```sh
+  cd my-project
+  npm init -y
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+npm start or live server
+
+
+```sh
+  GitHub Pages Server
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+
+```sh
+ npx stylelint "**/*.{css,scss}"
+```
+
+
+### Deployment
+
+You can deploy this project using:
+
+GitHub Pages
+
+```sh
+ https://vikas924.github.io/Hello-Microverse/
+```
+ 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Vikas Chib**
+
+- GitHub: [@vikas924](https://github.com/vikas924)
+- Twitter: [@vikas410190](https://twitter.com/vikas410190)
+- LinkedIn: [Vikas Singh](https://www.linkedin.com/in/vikas-singh-586a07132/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **[new_feature_1]**
+- [ ] **[new_feature_2]**
+- [ ] **[new_feature_3]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [https://github.com/vikas924/Hello-Microverse/issues](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Write a message to encourage readers to support your project
+
+If you like this project please follow me on github , twitter and also connect on Linkedin.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Give credit to everyone who inspired your codebase.
+
+I would like to thank Microverse for this exercise.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+- **[Question_1]**
+
+  - [Answer_1]
+
+- **[Question_2]**
+
+  - [Answer_2]
```

Comment:
-  [ ] For this section, you can include possible questions your viewers will be interested in or delete the section completely. 😃